app/src/main/java/org/greenstand/android/TreeTracker/capture/TreeCaptureViewModel.kt

```diff
@@ -43,6 +45,7 @@ class TreeCaptureViewModel(
     }
 
     suspend fun endSession() {
+        locationDataCapturer.stopGpsUpdates()
```

Comment:
I am not sure if sessionID is used in Tree Image Review would be better to call this in there in that case

Reply:
I'm not quite sure what you mean.

This is stopping GPS updates when the user ends the session. It can only be ended from the tree capture screen.